examples/federated-database-instance/README.md

```diff
@@ -0,0 +1,50 @@
+# Example - MongoDB Atlas Federated Database Instance
```

Comment:
```suggestion
# Example - MongoDB Atlas Federated Database Instance with AWS S3
```

@andreaangiolillo can you also update name in examples folder to clarify this example is for "federated-database-instance-AWS-S3"?

Reply:
Updated